cep47-logic/src/lib.rs

```diff
@@ -256,10 +244,8 @@ pub trait CEP47Storage {
     fn unpause(&mut self);
```

Comment:
Can you add comments to the methods of the trait? What is `pause` and `unpause`?

Reply:
done